src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  @include columns(1);
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    @include columns(2);
+  }
+
+  @media (min-width: 768px) {
+    @include columns(3);
+  }
+
+  @media (min-width: 1024px) {
+    @include columns(4);
+  }
```

Comment:
Look at this [slack message](https://mateacademy.slack.com/archives/C03H0MLSFUJ/p1655716733610009) and apply it in your code 😉